modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -30,66 +25,65 @@ yarn add @workday/canvas-kit-react
 
 ### Basic Example
 
-Below is a basic layout built with `Grid`. We created four components: `Header`, `SideBar`,
-`BodyContent` and `Footer`. We also used those names for their respective `gridArea` props. Then we
-can use those names in the parent component, `UIExample`, to organize the UI however we want. This
-allows the parent container to control how its children render. For page layouts, Workday typically
-uses a 12 column grid. In the example below, the Sidebar is 3 columns and the Body Content is 9
-columns.
+> **Note**: We recommend you familiarize yourself with
+> [CSS Grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids) before diving into
+> our `Grid` component. This example makes use of
+> [Grid Areas](https://developer.mozilla.org/en-US/docs/Glossary/Grid_Areas).
```

Comment:
The 12 column grid reference was suggested to us by Michael since it was from a design perspective. We can keep it or remove it, I'll leave it up to you.